internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -147,6 +147,11 @@ func resourceKustoCluster() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			"enable_auto_stop": {
```

Comment:
This is defaulted to true in the portal so we should Default it here too. It does complicate things as this value isn't returned in the api so we'll have to do a state migration to put this attribute as true into the statefile when people update their provider